schema.json

```diff
@@ -61,91 +61,7 @@
 		},
 
 		"configManagementConfig": {
-			"type": "object",
-			"oneOf": [
-				{
-					"required": ["helm"],
-					"additionalProperties": false,
-					"properties": {
-						"helm": {
-							"$ref": "#/definitions/helmConfig"
-						}
-					}
-				},
-				{
-					"required": ["kustomize"],
-					"additionalProperties": false,
-					"properties": {
-						"kustomize": {
-							"$ref": "#/definitions/kustomizeConfig"
-						}
-					}
-				},
-				{
-					"required": ["ytt"],
-					"additionalProperties": false,
-					"properties": {
-						"ytt": {
-							"$ref": "#/definitions/yttConfig"
-						}
-					}
-				}
-			]
-		},
-
-		"helmConfig": {
-			"type": "object",
-			"required": ["releaseName"],
-			"additionalProperties": false,
-			"properties": {
-				"releaseName": {
-					"type": "string",
-					"pattern": "^\\w[\\w-]*\\w$"
-				},
-				"namespace": {
-					"type": "string",
-					"pattern": "^\\w[\\w-]*\\w$"
-				},
-				"chartPath": {
-					"$ref": "#/definitions/relativePath"
-				},
-				"valuesPaths": {
-					"type": "array",
-					"items": {
-						"$ref": "#/definitions/relativePath"
-					}
-				}
-			}
-		},
-
-		"kustomizeConfig": {
-			"type": "object",
-			"additionalProperties": false,
-			"properties": {
-				"path": {
-					"$ref": "#/definitions/relativePath"
-				},
-				"enableHelm": {
-					"type": "boolean"
-				},
-				"loadRestrictor": {
-					"type": "string",
-					"enum": ["LoadRestrictionsRootOnly", "LoadRestrictionsNone"]
-				}
-			}
-		},
-
-		"yttConfig": {
-			"type": "object",
-			"additionalProperties": false,
-			"properties": {
-				"paths": {
-					"type": "array",
-					"items": {
-						"$ref": "#/definitions/relativePath"
-					}
-				}
-			}
+			"type": "object"
```

Comment:
Do we have some other method available to validate the config management object?

Even if so... I know how yucky it would be to reverse engineer a JSON schema from Argo CD, but what I really wanted with this schema is that we would eventually publish it somewhere public and then leverage it for context help.

If we have a decent schema, it would be easy, for instance, to create a VS Code plugin that recognizes `kargo-render.yaml` and provides context help as you edit.

Reply:
I've realized we can re-use application CRD definition. Added `./hack/fetch-argocd-schema.sh` that generates schema from CRD.

Reply:
Nice